### ISSUE TITLE
Bug: The value of binary (on-off) CC parameters cannot be changed from the Zynthian UI

### ISSUE THREAD

**Issue by polluxsynth** [zynthian/zynthian-issue-tracking#607]
**Describe the bug**
When parameters are CC controlled, if they are binary (on-off, i.e. with 2 states), it's not possible to change the value from the Zynthian UI. This is not a problem with continuous (0..127) CC parameters, or binary OSC parameters.

**To Reproduce**
1. Bring up a ZynAddSubFX layer.
2. Go to the portamento screen
3. Try to edit portamento on/off (which is mapped to CC 65).
4. Note how the value does not change.

**Expected behavior**
Turning the knob should result in the parameter value changing, but in actual fact, nothing happens.

**Setup:**
- SD image
2021-09-25-zynthianos-stable.img 
- HARDWARE: Zynthian kit v4.4 (4 GB)
Raspberry Pi 4 Model B Rev 1.4
Soundcard: HifiBerry DAC+ ADC PRO
Display: ZynScreen 3.5 (v1)
Wiring: MCP23017_ZynScreen
GPIO Expander: MCP23017
- SYSTEM
Raspbian GNU/Linux 10 (buster)
Build Date: 2021-09-25
Memory: 3% (121M/3839M)
SD Card: 44% (13G/29G)
Temperature: 48.2°C
- MIDI
Profile: default.sh
Fine Tuning: 440 Hz
Master Channel: 16
Single Active Channel: off
ZS3 SubSnapShots: on
- SOFTWARE
zyncoder: testing (558fd12)  
zynthian-ui: testing (f8d39ee)
zynthian-sys: testing (587dd76)
zynthian-data: testing (076a23d)
zynthian-webconf: testing (9a7956

**Comment by polluxsynth** [comment]
Created a PR for a suggested (but possibly not optimal) fix: https://github.com/zynthian/zynthian-ui/pull/418

**Comment by jofemodo** [comment]
Sorry. I can't reproduce this bug. I've tested on stable and testing and it works for me.
Please, test again and verify you are running latest testing or stable without extra "patches".

**Comment by polluxsynth** [comment]
It certainly was not on stable, originally I thought it was something I brought in when I brought in some patches from testing, but I didn't want to file a bug report based on that because it could very well have been the result of my particular patch set.  However, the other day I wanted to check out how jalv saves lv2 patches, so I switched all branches to testing, rebooted, upgraded, rebooted, and created a new snapshot with zynaddsubfx, and the problem was still there.

However, I did the same today, and now it works fine. I honestly can't explain it, the set of branches in the original report above were the ones I was running at the time and I haven't even upgraded since then so it's still the same commits.

So I'll close the issue; thank you @jofemodo for verifying that it works. I'll keep an eye out for this issue in case it happens again, and it's a more complex set of conditions that are the root cause.
